apollo-gradle-plugin/src/main/kotlin/com/apollographql/apollo/gradle/internal/DefaultApolloExtension.kt

```diff
@@ -15,13 +15,13 @@ open class DefaultApolloExtension(val project: Project)
   val services = mutableListOf<DefaultService>()
 
   /**
-   * compilationUnits is meant to be consumed by other gradle plugin.
+   * compilationUnit is meant to be consumed by other gradle plugin.
    * The apollo plugin will add the {@link CompilationUnit} as it creates them
    */
-  internal val compilationUnits = project.container(CompilationUnit::class.java)
+  internal val compilationUnit = project.container(CompilationUnit::class.java)
```

Comment:
This one is a container (which extends from Set) and internal. I think we should keep this one plural.

Reply:
+1